HTML/index.html

```diff
@@ -3,44 +3,219 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Movie website Home Page</title>
-  <link rel="stylesheet" href="../CSS/index.css">
-
+  <title>Movie</title>
+  <link rel="stylesheet" href="../CSS/style.css">
+  <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
 </head>
 <body>
-  <!-- Header Section -->
   <header>
-    <nav>
-      <!-- nav bar logo -->
-      <div class="logo">
-        <h1>Movie<span>360</span></h1>
-      </div>
-
-      <!-- Nav items: -->
-      <div class="nav-items">
+    <div class="logo">
+      <h1>Movie<span>360</span></h1>
+    </div>
+    <span class="close" id="close">&#10006;</span>
+    <span id="bars">
+      <i class='bx bx-menu'></i>
+    </span>
+    <nav id="nav">
+      
+      <div class="nav-items" id="nav-items">
         <ul>
-          <li><a href="">Home</a></li>
-          <li><a href="">About Us</a></li>
-          <li><a href="">Movies</a></li>
-          <li><a href="">Pages</a></li>
-          <li><a href="">Contact Us</a></li>
+          <li><a href="#">Home</a></li>
+          <!-- <li><a href="#">About</a></li> -->
+          <li><a href="#">Movies</a></li>
+          <li><a href="#">Tv Shows</a></li>
+          <li><a href="#">Pricing</a></li>
+          <li><a href="#">Blog</a></li>
+          <!-- <li><a href="#">Contacts</a></li> -->
         </ul>
       </div>
-      <div class="search-bar">
-        <input type="text" placeholder="Search For Movies Here..." id="SearchInput">
-        <button id="SearchBtn">Search</button>
-      </div>
+     
+        <div class="search">
+          <input type="text" placeholder="Search Movie..." id="search">
+          <button id="btn">Search</button>
+        </div>
+
     </nav>
   </header>
 
-  <div class="row" id="movie-box">
+  <section class="main" id="main-container">
+   
+<div class="main1">
+    <div class="text" id="text-container">
+      <h1>Movie<span>360</span></h1>
+      <p>Unlimited <span>Movie</span>, tv shows, & more</p>
+      <button>watch now</button>
+    </div>
+    <div class="images">
+      <img src="../images/img4.jpg" alt="">
+    </div>
+    </div>
+  </section>
+  <div class="container">
+    
+    <div class="movies-container favorites">
+
+    
+        <h1 class="title">Favorites</h1>
+        <div class="slider-wrapper">
+         <div class="movies-grid">
+         </div>
+      </div>
+    </div>
+
+    <div class="movies-container trending">
+        <h1>Trending</h1>
 
+        <div class="slider-wrapper">
+          <div class="movies-grid">
+          </div>
+       </div>
+    </div>
+
+    <!-- <div class="movies-container series">
```

Comment:
Bad practice to leave codes commented for review unless very necessary. If you do not need it, delete it.